src/index.html

```diff
@@ -1,14 +1,323 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
-    <link rel="stylesheet" href="./styles/main.css">
-  </head>
-  <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dia</title>
+  <link rel="icon" href="./images/logo/icon.png">
+  <link rel="stylesheet" href="./styles/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <header class="header" id="home">
+    <img src="./images/logo/logo.svg" alt="logo" class="header__logo">
+    <input type="checkbox" class="header__nav-toggler" id="nav-toggler"/>
+    <label class="header__nav-toggler-label" for="nav-toggler"></label>
+    <nav class="nav header__nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a href="#home" class="nav__link">Home</a>
+        </li>
+        <li class="nav__item">
+          <a href="#about" class="nav__link">About us</a>
+        </li>
+        <li class="nav__item">
+          <a href="#services" class="nav__link">Services</a>
+        </li>
+        <li class="nav__item">
+          <a href="#work" class="nav__link">Work</a>
+        </li>
+        <li class="nav__item">
+          <a href="#contact" class="nav__link">Contact us</a>
+        </li>
+        <li class="nav__item nav__item-active" tabindex="0">Hire us</li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="info-section">
+      <div class="info">
+        <h1 class="info__heading">Global digital agency</h1>
+        <p class="info__text">We believe in the power of bold ideas that can solve business challenges.</p>
```

Comment:
переноси для читаемости
```suggestion
        <p class="info__text">
          We believe in the power of bold ideas that can solve business challenges.
        </p>
```